types/index.d.ts

```diff
@@ -73,6 +73,7 @@ export interface RenderOptions<
    */
   hydrate?: boolean
   /**
+   * Only works if used with React 18.
```

Comment:
Should we also add this to our docs?

Reply:
yep

Reply:
source: https://github.com/testing-library/testing-library-docs/pull/1380
preview: https://deploy-preview-1380--testing-library.netlify.app/docs/react-testing-library/api#legacyroot